File: result/repair/transcoder/python2java/java_repair/REMOVE_MINIMUM_ELEMENTS_EITHER_SIDE_2MIN_MAX.java
  int f_gold ( int [ ] arr , int n ) {
  int longestStart = - 1 ;
  ;
  int longestEnd = 0 ;
  for ( int start = 0 ;
  start < n ;
  start ++ ) {
    int min = Integer . MAX_VALUE ;
    int max = - Integer . MAX_VALUE ;
    for ( int end = start ;
    end < n ;
    end ++ ) {
      int val = arr [ end ] ;
      if ( ( val < min ) && ( val > max ) ) min = val ;
      if ( ( val < 2 * min ) && ( val > max ) ) max = val ;
      if ( 2 * min <= max ) break ;
      if ( ( end - start > longestEnd - longestStart || longestStart == - 1 ) ) {
        longestStart = start ;
        longestEnd = end ;
      }
    }
  }
  if ( longestStart == - 1 ) return n ;
  return ( n - ( longestEnd - longestStart + 1 ) ) ;
}

File: result/repair/transcoder/java2python/python_repair/REMOVE_MINIMUM_ELEMENTS_EITHER_SIDE_2MIN_MAX.py
def f_gold ( arr , n ) :
    longest_start , longest_end = - 1 , 0
    for start in range ( n ) :
        min , max = sys.maxsize, -sys.maxsize
        for end in range ( start , n ) :
            val = arr [ end ]
            if val < min : min , val = val , val
            if val > max : max , val = val , val
            if 2 * min <= max : break
            if end - start > longest_end - longest_start or longest_start == - 1 :
                longest_start = start
                longest_end = end
    if longest_start == - 1 :
        return n
    return ( n - ( longest_end - longest_start + 1 ) )

